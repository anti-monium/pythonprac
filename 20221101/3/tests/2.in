a = Grange(1, 7, 10000000)
b = Grange(1, 4, 1)
c = Grange(1, 5, 4)
print(len(b[1:1000]))
print(len(a), *list(a))
print(len(c[1:1111111:2]))
if bool(b):
	print('There is b')
else:
	print('Who stole the b!?')
print(b, *list(b), c, *list(c))
print(a[7])
