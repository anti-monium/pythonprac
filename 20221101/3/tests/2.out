5
9 1 7 49 343 2401 16807 117649 823543 5764801
5
Who stole the b!?
grange(1, 4, 1) grange(1, 5, 4) 1
823543
